js_modules/dagster-ui/packages/ui-core/src/runs/LogsRowStructuredContent.tsx

```diff
@@ -518,22 +523,30 @@ const AssetMetadataContent = ({
     </span>
   );
 
+  // Note: No memoization here - log row content components are memoized higher up
+
+  const rows: LogRowStructuredRow[] = [
+    {
+      label: 'asset_key',
+      item: (
+        <>
+          {displayNameForAssetKey(assetKey)}
+          {assetDashboardLink}
+        </>
+      ),
+    },
+  ];
+
+  if (partition) {
+    rows.push({label: 'partition', item: <>{partition}</>});
+  }
```

Comment:
nbd but I do prefer memoizing defensively rather than relying on parent component memoization

Reply:
Yeah I think these log row "content" components should probably all get memos in that case, let me see if i can do it in a follow-up